### PR TITLE
Update links

### PR DESCRIPTION
**Pull request by lisicase**
Updates all link schemes of working links from "http" links to "https" and a few other old links.

### CODE REVIEW

**Review by xuzhg — APPROVED** [review]

File: _ecosys/consumers/Pebble-Reports.md [hunk 1]
```diff
@@ -2,6 +2,6 @@
 layout: default
 category: consumers
 title: "Pebble Reports"
-link: "http://pebblereports.com/odata/"
+link: "https://pebblereports.com/odata/"
```
Comment (mikepizzo) [inline _ecosys/consumers/Pebble-Reports.md:5]: this link works for me as http: but not as https: ("The Connection was Reset")

File: _ecosys/consumers/Teiid.md [hunk 2]
```diff
@@ -2,6 +2,6 @@
 layout: default
 category: consumers
 title: "Teiid"
-link: "http://teiid.jboss.org/"
+link: "https://teiid.jboss.org/"
```
Comment (mikepizzo) [inline _ecosys/consumers/Teiid.md:5]: Should move to new site: [https://teiid.io/](https://teiid.io/)

File: _ecosys/consumers/Telerik-RadControls-for-Silverlight-and-WPF.md [hunk 3]
```diff
@@ -2,6 +2,6 @@
 layout: default
 category: consumers
 title: "Telerik RadControls for Silverlight and WPF"
-link: "http://www.telerik.com/products/odata-support.aspx"
+link: "https://www.telerik.com/products/odata-support.aspx"
```
Comment (mikepizzo) [inline _ecosys/consumers/Telerik-RadControls-for-Silverlight-and-WPF.md:5]: It looks like this has been discontinued and we should probably delete this entry.

File: _ecosys/consumers/Telerik-RadGrid-for-ASP.NET-Ajax.md [hunk 4]
```diff
@@ -2,6 +2,6 @@
 layout: default
 category: consumers
 title: "Telerik RadGrid for ASP.NET Ajax"
-link: "http://www.telerik.com/products/odata-support.aspx"
+link: "https://www.telerik.com/products/odata-support.aspx"
```
Comment (mikepizzo) [inline _ecosys/consumers/Telerik-RadGrid-for-ASP.NET-Ajax.md:5]: It looks like this has been discontinued and we should probably delete this entry.

File: _ecosys/consumers/Telerik-Reporting.md [hunk 5]
```diff
@@ -2,6 +2,6 @@
 layout: default
 category: consumers
 title: "Telerik Reporting"
-link: "http://www.telerik.com/products/odata-support.aspx"
+link: "https://www.telerik.com/products/odata-support.aspx"
```
Comment (mikepizzo) [inline _ecosys/consumers/Telerik-Reporting.md:5]: It looks like this has been discontinued and we should probably delete this entry.

File: _ecosys/consumers/Telerik.md [hunk 6]
```diff
@@ -2,6 +2,6 @@
 layout: default
 category: consumers
 title: "Telerik"
-link: "http://www.telerik.com/products/odata-support.aspx"
+link: "https://www.telerik.com/products/odata-support.aspx"
```
Comment (mikepizzo) [inline _ecosys/consumers/Telerik.md:5]: It looks like this has been discontinued and we should probably delete this entry.

File: _ecosys/liveservices/Telerik.md [hunk 7]
```diff
@@ -2,6 +2,6 @@
 layout: default
 category: liveservices
 title: "Telerik"
-link: "http://www.telerik.com/products/odata-support.aspx"
+link: "https://www.telerik.com/products/odata-support.aspx"
```
Comment (mikepizzo) [inline _ecosys/liveservices/Telerik.md:5]: It looks like this has been discontinued and we should probably delete this entry.

File: _ecosys/liveservices/Telerik-TV.md [hunk 8]
```diff
@@ -2,6 +2,6 @@
 layout: default
 category: liveservices
 title: "Telerik TV"
-link: "http://tv.telerik.com/services/odata.svc"
+link: "https://tv.telerik.com/services/odata.svc"
```
Comment (mikepizzo) [inline _ecosys/liveservices/Telerik-TV.md:5]: I got an error on this page.

File: _ecosys/liveservices/Windows-Live.md [hunk 9]
```diff
@@ -2,6 +2,6 @@
 layout: default
 category: liveservices
 title: "Windows Live"
-link: "http://live.com/"
+link: "https://live.com/"
 ---
```
Comment (mikepizzo) [inline _ecosys/liveservices/Windows-Live.md:6]: We should really update this to be microsoft graph.

File: _ecosys/producers/GeoREST.md [hunk 10]
```diff
@@ -2,6 +2,6 @@
 layout: default
 category: producers
 title: "GeoREST"
-link: "http://code.google.com/p/georest/"
+link: "https://code.google.com/p/georest/"
```
Comment (mikepizzo) [inline _ecosys/producers/GeoREST.md:5]: I get an error for this url, but this one works: [https://code.google.com/archive/p/georest/](https://code.google.com/archive/p/georest/)

File: _ecosys/producers/Teiid.md [hunk 11]
```diff
@@ -2,6 +2,6 @@
 layout: default
 category: producers
 title: "Teiid"
-link: "http://teiid.jboss.org/"
+link: "https://teiid.jboss.org/"
```
Comment (mikepizzo) [inline _ecosys/producers/Teiid.md:5]: should update to [https://teiid.io/](https://teiid.io/)

File: _ecosys/tutorials/Tableau.md [hunk 12]
```diff
@@ -2,6 +2,6 @@
 layout: default
 category: tutorials
 title: "Tableau"
-link: "http://onlinehelp.tableau.com/current/pro/online/windows/en-us/help.htm#examples_odata.html"
+link: "https://onlinehelp.tableau.com/current/pro/online/windows/en-us/help.htm#examples_odata.html"
```
Comment (mikepizzo) [inline _ecosys/tutorials/Tableau.md:5]: update to [https://help.tableau.com/current/pro/desktop/en-us/examples_odata.htm](https://help.tableau.com/current/pro/desktop/en-us/examples_odata.htm)

File: _libraries/ASP-NET-Web-API-OData.md [hunk 13]
```diff
@@ -2,11 +2,11 @@
 category: net
 name: ASP.NET Web API OData
```
Comment (mikepizzo) [inline _libraries/ASP-NET-Web-API-OData.md:3]: we should add an entry for ASP.NET Core OData

**Comment by mikepizzo** [comment]
> version: V4 (4.0.0-beta-01)

this is released. Should remove "-beta" from title and line 3, and (4.0.0-beta-01) from version on line 5.

---
Refers to: _libraries/ODataJS-beta.md:5 in 8cb3b00. [](commit_id = 8cb3b00e0b69d2b9f1aa17eb1e95b0ed71233a64, deletion_comment = False)

File: _libraries/RESTier.md [hunk 14]
```diff
@@ -2,11 +2,11 @@
...
 name: RESTier
 rownumber: 43
-link: http://odata.github.io/RESTier/
+link: https://odata.github.io/RESTier/
 version: V4
 object: Server
 downloads:
   - source: nuget gallery
-    link: http://www.nuget.org/packages/Microsoft.Restier/
+    link: https://www.nuget.org/packages/Microsoft.Restier/
```
Comment (mikepizzo) [inline _libraries/RESTier.md:10]: update link to: [https://www.nuget.org/packages/Microsoft.Restier.AspNet/](https://www.nuget.org/packages/Microsoft.Restier.AspNet/)

File: _libraries/ASP-NET-Web-API-OData.md (hunk 13 above)
Comment (lisicase) [reply _libraries/ASP-NET-Web-API-OData.md:3]: The link for Web API forwards to https://learn.microsoft.com/en-us/odata/. Do we want that to be the link for ASP.NET Core OData as well, or something else?

Comment (mikepizzo) [reply _libraries/ASP-NET-Web-API-OData.md:3]: Let's use https://learn.microsoft.com/en-us/odata/webapi-8/Overview for Asp.net Core OData, and https://learn.microsoft.com/en-us/odata/webapi/getting-started for Web API.

File: _ecosys/liveservices/Telerik-TV.md [hunk 15]
```diff
@@ -4,4 +4,4 @@ category: liveservices
 title: "Telerik TV"
 link: "http://tv.telerik.com/services/odata.svc"
 ---
-Telerik’s catalog of libraries, videos, Tags and Series
+Telerik�s catalog of libraries, videos, Tags and Series
```
Comment (mikepizzo) [inline _ecosys/liveservices/Telerik-TV.md:7]: >� [](http://example.com/codeflow?start=7&length=1)

Is this change intended?

File: _libraries/ASP-NET-Web-API-OData.md (hunk 13 above)
Comment (mikepizzo) [reply _libraries/ASP-NET-Web-API-OData.md:3]: I don't see the new link for ASP.NET Core OData -- is that in a new file that needs to be added to the PR?

**Review by mikepizzo — CHANGES_REQUESTED** [review]
:clock1:

File: _ecosys/liveservices/Windows-Live.md (hunk 9 above)
Comment (mikepizzo) [reply _ecosys/liveservices/Windows-Live.md:6]: Addressed in PR#338

**Comment by mikepizzo** [comment]
> version: V4 (4.0.0-beta-01)

Addressed in PR#338

---
In reply to: [1377780478](https://github.com/OData/odataorg.github.io/pull/322#issuecomment-1377780478) [](http://example.com/codeflow?ancestors=1377780478,1377780478)

---
Refers to: _libraries/ODataJS-beta.md:5 in 8cb3b00. [](commit_id = 8cb3b00e0b69d2b9f1aa17eb1e95b0ed71233a64, deletion_comment = False)

File: _ecosys/liveservices/Telerik-TV.md (hunk 15 above)
Comment (lisicase) [reply _ecosys/liveservices/Telerik-TV.md]: It was not on purpose and should be fixed now.

**Review by mikepizzo — APPROVED** [review]
:shipit:
